RecoEgamma/PhotonIdentification/data/Fall17/effAreaPhotons_cone03_pfNeutralHadrons_90percentBased_TrueVtx.txt

```diff
@@ -0,0 +1,16 @@
+# This file contains Effective Area constants for 
+# computing pile-up corrections for the neutral hadron isolation
+# for a photon object.
+#
+#  The effective areas are derived based on the 90% contour method.
+# These effective areas are also found on the following slides:
+#      https://indico.cern.ch/event/491517/contributions/2349134/attachments/1359450/2056689/CutBasedPhotonID_24-10-2016.pdf
+#
+# |eta| min   |eta| max   effective area
+0.0000         1.0000        0.064
+1.0000         1.4790        0.110
+1.4790         2.0000        0.076
+2.0000         2.2000        0.023
+2.2000         2.3000        0.0151
+2.3000         2.4000        0.0001
```

Comment:
[reposting in the updated file thread]
0.0001 looks wrong.
What would be the reason for a factor of 100 drop in neutral hadrons response in one single bin?

Reply:
Numbers are correct, are presented here: https://indico.cern.ch/event/662751/contributions/2778043/attachments/1562017/2459674/EGamma_WorkShop_21.11.17_Debabrata.pdf

Reply:
(My two cents): I would check that particular bin, because it is suspiciously too small with respect to all others. If any problem/error with the fit is discovered there, perhaps a more reasonable estimate could be the average value of the two nearby bins, i.e. about 0.014

Reply:
Analyzer working on this. Will show update at next EGM meeting. I`ll add updates here after that.

Reply:
thanks.

Reply:
@lsoffi 
BTW, what will be the implication if this is actually a bug?

My guess is to correct for it, at least some cuts or scale factors downstream will need to be fixed. Would anything else need a change?

Depending on the answers, I guess the "bug" can then be qualified as a feature to be fixed in the next update or, alternatively, something to be updated with this PR.

Reply:
Dear @slava77 , indeed i do not think this is a bug. This is definitely a feature that can be improved in the next iteration.